README.md

```diff
@@ -11,16 +11,14 @@ This module implements the Thin Shell Demons regularization proposed in
 > MIUA 2015  
 
 
-There are two implementations of this approach available:
-1. (TSHDv4) An implementation taking advantage of the point set to point set registration
-   capabilities of the ITK v4 registration framework.
-2. (TSHD) An implementation started by Qingyu Zhao and updated to fix some remaining task
-   and run with ITK 5
 
 > :warning: **This module requires to be compiled against an ITK version with**  
 > - [PointSetToPointSetMetricWithIndexv4](https://github.com/InsightSoftwareConsortium/ITK/pull/2385)   
 > 
-> If you are compiling against an older version  use the ones inlcuded in this repository marked with h/hxx-unused (remove the -unused).
+
+<p align="center">
+<img src="https://user-images.githubusercontent.com/1044135/158479969-7313ed94-c5fb-4803-ae8d-2c0631893664.png" width="400" height="300">
```

Comment:
Would it be possible to include a description for the image, you could something like this : 

| <img src="https://user-images.githubusercontent.com/1044135/158479969-7313ed94-c5fb-4803-ae8d-2c0631893664.png" width="400" height="300"> |
|--|
| Left: This is ..., Right: This is ...|

```
| <img src="https://user-images.githubusercontent.com/1044135/158479969-7313ed94-c5fb-4803-ae8d-2c0631893664.png" width="400" height="300"> |
|--|
| Left: This is ..., Right: This is ...|
```

Reply:
Yes I will add it. I am also getting results on the mouse dataset and will add them here too.